doc/rst/platforms/cray.rst

```diff
@@ -59,16 +59,33 @@ module.
 To use Chapel with the default settings and confirm it is correctly
 installed, do the following:
 
-1) Load the Chapel module::
+1) Load the Chapel module:
 
-     module load chapel
+   If you are using the Tcl module system, first load these required
```

Comment:
How would an average user know if they're on tcl/lmod?

Reply:
So far as I know the tcl module will load mpich and PrgEnv-gnu for the user if they haven't, so I don'w know that we need these for tcl

Reply:
The output of `module --version` is quite different, though I'm not sure we want to get into that here, or at least not yet.  (For Tcl it starts with `VERSION=<dottedNumber>` while for Lmod it starts with `Modules based on Lua: Version <dottedNumber>`.)  An alternative would be for them to check the value of `$MODULESHOME`, which starts with `/opt/cray/pe/modules/` for Tcl and is `/usr/share/lmod/lmod` for Lmod.  But I don't know how dependable that is.

Reply:
Offline we agreed to go with this for now, and will improve over time as-needed